lldb/test/Shell/Target/target-label.test

```diff
@@ -4,16 +4,16 @@
 
 target create -l "ls" /bin/ls
 target list
-# CHECK: * target #0 (ls): /bin/ls
+# CHECK: * target #0 (ls): [[LS_PATH:.*]]
```

Comment:
> Do I understand correctly here that the path `/bin/ls` is given to target create, that program is sent to the remote and this LS_PATH will be `<platform working dir>/ls`? So that's why the substitution is needed.

Yes.

> Though /bin/ls surely isn't going to be compatible with the remote a large percentage of the time. I assume it's never actually run.

Yes, it only tests labels.


> Could make this supported on Windows too if we happen to have some basic C file hanging around in this dir and use that to make the test binaries instead, but out of scope for this PR.

We can use util names without paths here (to load them from PATH). lldb on Windows will handle that, but I don't know if @medismailben had reasons to use absolute paths.

Reply:
Sorry, I was wrong here.

`target create` command accepts remote path. But `target list` prints full path to host's cached copy of the executable:
```
(lldb) target list
Current targets:
* target #0: C:\Users\vdzhidzhoev\.lldb\module_cache\remote-linux\.cache\16852127-4D24-D03C-BAB1-365C28DC2007-8BF46835\ls ( arch=aarch64-*-linux, platform=remote-linux )
```

So we can't just place 'ls' in the pattern matching line.